Angular Admin UI compilation fails in Vendure 3.5.2 due to missing package exports

**Describe the bug**
After upgrading from Vendure 3.5.1 to 3.5.2, the Angular Admin UI compilation fails with module resolution errors. The `@vendure/admin-ui` package's `exports` field in `package.json` only exports the root path (`.`), but the compiled Admin UI extensions attempt to import from subpaths like `@vendure/admin-ui/core`, `@vendure/admin-ui/login`, etc., which are no longer exported.

Vendure: 3.5.2 (upgraded from 3.5.1)
Node: v22.12.0
npm: 11.5.2
Angular: 19.2.4
Error count: 200+ compilation errors
Root cause: The exports field in node_modules/@vendure/admin-ui/package.json only includes: 
"exports": {
    ".": { ... }
}

**To Reproduce**
## Package Versions
```json
{
  "@vendure/admin-ui": "^3.5.2",
  "@vendure/admin-ui-plugin": "3.5.2",
  "@vendure/ui-devkit": "3.5.2",
  "@vendure/core": "3.5.2",
  "@angular-devkit/build-angular": "^19.2.10",
  "@angular/animations": "^19.2.9",
  "@angular/platform-browser-dynamic": "^19.2.9"
}
```

**Expected behavior**
A clear and concise description of what you expected to happen.

**Actual behavior**
✘ [ERROR] Could not resolve "@vendure/admin-ui/core"

    src/app.module.ts:4:61:
      4 │ ...nt, AppComponentModule, CoreModule } from '@vendure/admin-ui/core';
        ╵                                              ~~~~~~~~~~~~~~~~~~~~~~~~

  The path "./core" is not exported by package "@vendure/admin-ui":

    ../node_modules/@vendure/admin-ui/package.json:27:15:
      27 │     "exports": {
         ╵                ^

  You can mark the path "@vendure/admin-ui/core" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
**Screenshots/Videos**
If applicable, add screenshots or videos to help explain your problem.

--- a/.github/workflows/publish_to_npm.yml
+++ b/.github/workflows/publish_to_npm.yml
@@ -127,17 +127,22 @@ jobs:
       # Publish to npm with appropriate dist-tag
       # Note: we do not use `lerna publish` because at the time of writing, it fails to publish
       # using the Trusted Publishing workflow for some reason.
+      # We use a custom script to handle publishConfig.directory for packages like @vendure/admin-ui
+      # that need to publish from a nested directory.
       - name: Publish to NPM (release)
         if: matrix.type == 'release'
-        run: npx lerna exec --no-private -- npm publish --access public
+        run: |
+          npx lerna exec --no-private -- 'publish_dir=$(node -p "require(\"./package.json\").publishConfig?.directory || \".\"") && cd "$publish_dir" && npm publish --access public'
 
       - name: Publish to NPM (master-nightly)
         if: matrix.type == 'master-nightly' && steps.commit_check.outputs.should_publish == 'true'
-        run: npx lerna exec --no-private -- npm publish --access public --tag master
+        run: |
+          npx lerna exec --no-private -- 'publish_dir=$(node -p "require(\"./package.json\").publishConfig?.directory || \".\"") && cd "$publish_dir" && npm publish --access public --tag master'
 
       - name: Publish to NPM (minor-nightly)
         if: matrix.type == 'minor-nightly' && steps.commit_check.outputs.should_publish == 'true'
-        run: npx lerna exec --no-private -- npm publish --dd --access public --tag minor
+        run: |
+          npx lerna exec --no-private -- 'publish_dir=$(node -p "require(\"./package.json\").publishConfig?.directory || \".\"") && cd "$publish_dir" && npm publish --access public --tag minor'
 
       - name: Skip publish (no new commits)
         if: matrix.type != 'release' && steps.commit_check.outputs.should_publish == 'false'

